<ui version="4.0" >
 <class>AddFdbkDlg</class>
 <widget class="QWidget" name="AddFdbkDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>User</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbUsers" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2" >
     <item>
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>Profile</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbProfiles" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_3" >
     <item>
      <widget class="QLabel" name="textLabel3" >
       <property name="text" >
        <string>Assessment</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbFdbk" >
       <item>
        <property name="text" >
         <string>Relevant</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Fuzzy Relevant</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Irrelevant</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
